source/features/show-user-top-repositories.tsx

```diff
@@ -16,7 +16,7 @@ function buildUrl(queryField: string): URL {
 
 function init(): void {
 	// Showcase title
-	select('.js-pinned-items-reorder-container .text-normal')?.firstChild!.after(
+	select('.js-pinned-items-reorder-container .text-normal')!.firstChild!.after(
```

Comment:
I think this was necessary. Profiles might not have pinned items

Reply:
Seems not https://github.com/goastat